problems/objects.js

```diff
@@ -141,7 +188,17 @@ const addsCountry = () => {};
  * @returns {Object} {"Mark Twain": 8.9, "Nathaniel Hawthorne": 5.1}
  */
 
-const authorScores = () => {};
+const authorScores = (authors) => {
+  let authorScoreObj = {};
+  for (let i = 0; i < authors.length; i++) {
+    let el1 = authors[i][0];
```

Comment:
I love that you're breaking these out can improve them by choosing better variable names and declaring them with const ? 
author and score